src/fci/fci.cc

```diff
@@ -35,7 +35,8 @@
 
 #include "fci.h"
 
-namespace psi {
+using namespace psi;
```

Comment:
Keep namespace here?

Reply:
Here I am using the namespace. This allows me to avoid using `psi::outfile->` everywhere. To remove the name space completely we would have to isolate Forte from psi4.